frontend/test/metabase-lib/lib/Dimension.unit.spec.js

```diff
@@ -359,7 +387,8 @@ describe("Dimension", () => {
           expect(fieldInfo).toEqual(OVERWRITTEN_USER_ID_FIELD_METADATA);
         });
 
-        it("should not merge regular question's field results metadata with field info", () => {
+        // TODO: confirm that we don't need to worry about this
+        it.skip("should not merge regular question's field results metadata with field info", () => {
```

Comment:
Should we merge with this new skip, or confirm first?

Reply:
Good catch -- I forgot to address this. Let me figure out what the implications of this are.

Reply:
OK, I've limited some logic to models/datasets so that this unit test passes. I think it'd probably be fine to open this logic up to all questions, but we might as well limit this to models for now in order to avoid new regressions. We will need to open this logic up to metrics as well, perhaps via a more general `question.canOverrideMetadata` method or something.